release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -738,6 +756,14 @@ In {product-title} 4.7, the Insights Operator collects the following additional
 
 With this additional information, Red Hat can provide improved remediation steps in {cloud-redhat-com}.
 
+[id="ocp-4-7-auth"]
+=== Authentication and authorization
+
+[id="ocp-4-7-auth-tp-aws-sts"]
+==== Support for Amazon Web Services Security Token Service (Technology Preview)
```

Comment:
@joelddiaz plz give :+1: / :-1:

Reply:
I would word it more like "Running OpenShift using AWS Security Token Service (STS) for credentials (Technology Preview)"